python/spacewalk/rhn-conf/rhn_server_satellite.conf

```diff
@@ -43,3 +43,9 @@ default_mail_from =
 reposync_download_threads = 5
 reposync_timeout = 300
 reposync_minrate = 1000
+
+# URLGrabber log level. This parameter is used by spacewalk-repo-sync to provide
+# additional logs, overriding URLGRABBER_DEBUG. It takes the form "level,filename". 
+# Commonly, to debug spacewalk-repo-sync, use urlgrabber_logspec = 1,- : it will set
+# debug log and use reposync log as output.
+urlgrabber_logspec =
```

Comment:
I'm now wondering if it makes sense to have different log settings for urlgrabber than we have for the rest of reposync. What do you think about setting the `logspec` argument to "1,-" whenever reposync is executed in debug mode without making this configurable for the user?

I think it would make debugging easier if there were fewer things to tweak, but I'm curios to hear others' opinions :smiley: